src/machine/machined-varlink.c

```diff
@@ -437,14 +464,17 @@ static JSON_DISPATCH_ENUM_DEFINE(json_dispatch_acquire_metadata, AcquireMetadata
 static int vl_method_list(sd_varlink *link, sd_json_variant *parameters, sd_varlink_method_flags_t flags, void *userdata) {
         static const sd_json_dispatch_field dispatch_table[] = {
                 VARLINK_DISPATCH_MACHINE_LOOKUP_FIELDS(MachineLookupParameters),
+                { "acquireMetadata", SD_JSON_VARIANT_STRING, json_dispatch_acquire_metadata, offsetof(MachineLookupParameters, acquire_metadata), 0 },
                 VARLINK_DISPATCH_POLKIT_FIELD,
                 {}
         };
 
         Manager *m = ASSERT_PTR(userdata);
         _cleanup_(machine_lookup_parameters_done) MachineLookupParameters p = {
                 .pidref = PIDREF_NULL,
+                .acquire_metadata = _ACQUIRE_METADATA_INVALID,
```

Comment:
this should probably default to ACQUIRE_METADATA_NO, no? i.e. if clients don't ask for it we probably should not bother, by default.